clients/fides-js/src/lib/initialize.ts

```diff
@@ -361,6 +362,11 @@ export const initialize = async ({
     });
   }
 
+  // Call extensions
+  // DEFER(PROD#1439): This is likely too late for the GPP stub.
+  // We should move stub code out to the base package and call it right away instead.
+  await setupExtensions(options);
```

Comment:
I just changed this to `await` because otherwise `FidesInitialized` might be called before the extension is ready to listen for it. I'm not sure if this is okay though in terms of initialization times—the first `FidesInitialized` from an existing cookie will already have fired, so this is counting on the second one to fire. It would happen after the UI initializes, which might be far enough away from initial load times?